FlappyBird.py

```diff
@@ -347,18 +353,34 @@ def GameOver():
         clock.tick(60)
 
 elif choice == '3':
-    def Sorting_scores(e):
-        return int(e[1])
-    highscore_file = open("Highscores.txt", "r+")
-    index = 0
-    scores = []
-    for i in highscore_file.readlines():
-        scores.append(i.split())
-        index = index+1
-
-    scores.sort(reverse=True, key=Sorting_scores)
-    print("\n-----------------------HighScores-----------------------")
-    for index in scores:
-        print(index[0] + " - " + index[1])
-    input("\nPress any key to exit")
-
+    print("\n1.common highscores \n2.individual highscores")
+    ch = int(input())
+    if ch == 1:
+        def Sorting_scores(e):
+            return int(e[1])
+        highscore_file = open("Highscores.txt", "r+")
+        index = 0
+        scores = []
+        for i in highscore_file.readlines():
+            scores.append(i.split())
+            index = index+1
+
+        scores.sort(reverse=True, key=Sorting_scores)
+        print("\n-----------------------HighScores-----------------------")
+        for index in scores:
+            print(index[0] + " - " + index[1])
+        input("\nPress any key to exit")
+    
```

Comment:
Try to make it a set so that no same names are repeated. There should be only one highscore of a user in the global highscore tally. Try fixing it. Best of luck! 😀